acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -136,7 +136,7 @@ public void startNode(final BesuNode node) {
       }
 
       if (node.getPrivacyParameters().isOnchainPrivacyGroupsEnabled()) {
-        params.add("--privacy-onchain-groups-enabled");
+        params.add("--privacy-flexible-groups-enabled");
```

Comment:
for this we are going to have to deprecate the existing option and keep it for at least a major release (a quarter)